homeassistant/components/rointe/climate.py

```diff
@@ -0,0 +1,275 @@
+"""Support for Rointe Climate."""
+
+from __future__ import annotations
+
+from rointesdk.device import RointeDevice
+
+from homeassistant.components.climate import (
+    PRESET_COMFORT,
+    PRESET_ECO,
+    ClimateEntity,
+    ClimateEntityDescription,
+    ClimateEntityFeature,
+    HVACAction,
+    HVACMode,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import TEMP_CELSIUS
```

Comment:
Per the Actions warning, this constant is deprecated by the `UnitOfTemperature` enum.